content/basics/templates/pseudo-parameters/index.md

```diff
@@ -219,10 +219,9 @@ First, under the _Parameters_ section, add a template parameter `S3BucketNamePre
 S3BucketNamePrefix:
   Description: The prefix to use for your S3 bucket
   Type: String
-  Default: my-demo-bucket
-  AllowedPattern: ^[0-9a-zA-Z]+([0-9a-zA-Z-]*[0-9a-zA-Z])*$
+  Default: cfn-workshop
+  AllowedPattern: ^(?!(^xn--|.$))^[a-z0-9][a-z0-9-]{1,61}[a-z0-9]$
```

Comment:
Why is this different to the other instances?

Reply:
This is as the name suggests prefix only where we want to restrict xn-- and . The Sub than appends the bucket name with region and account ID.